lib/nano.js

```diff
@@ -172,6 +172,16 @@ module.exports = exports = function dbScope (cfg) {
       body = { message: body }
     }
 
+    if (!body && req.signal) {
```

Comment:
`req.signal.aborted` should be `true` also, but (at least in the tests) this doesn't seem to occur.